docs/installation/development.md

```diff
@@ -20,19 +20,75 @@ Tilt is a great way to run OpenCost on a local or remote Kubernetes environment.
-4. If using `kind`, in a terminal, run `kind create cluster` to create local cluster, or set your Kubernetes context to the desired cluster.
-5. In a terminal, run `tilt up -- --arch [amd64 or arm64] --docker-repo [repo or username]` at the root of the OpenCost repository
-        `--arch amd64` will compile OpenCost for amd64 nodes, `--arch arm64` will compile for arm64 nodes
-        `--docker_repo` is only required for remote development, or you want the images saved in Docker Hub. The value is prepended to the image tag (the -t flag in a Docker build command). [See Docker Documentation](https://docs.docker.com/engine/reference/commandline/build/#tag).
-        Example: `tilt up -- --arch amd64 --docker-repo example` would build OpenCost for a Kubernetes node with an `amd64` architecture, and push the images to the Docker Hub user `example`.
-6. Follow instructions to open the Tilt UI in browser (hit space, or navigate to http://localhost:10350)
-7. As you update your code or `tilt-values.yaml`, Tilt will automatically watch relevant source code and recompile, build, deploy, and run tests.
-8. `tilt down` will tear down resources created, otherwise they will continue to run indefinitely.
-9. `kind delete` cluster will delete the local cluster, if desired.
+2. Update `tilt-values.yaml` or create a new yaml file to contain desired attributes - [see Helm chart.](https://github.com/opencost/opencost-helm-chart/blob/main/charts/opencost/README.md)
+3. Optional - configure JSON files for cloud integration(s) and a service key [(see configuration documentation for more).](../configuration)
+4. Update `Tiltfile` to allow your remote Kubernetes contexts, if desired. See comment block near top of the `Tiltfile`. [See tilt documentation here.](https://docs.tilt.dev/api#api.allow_k8s_contexts)
+5. If using `kind`, in a terminal, run `kind create cluster` to create local cluster, or set your Kubernetes context to the desired cluster.
+6. Configure a `tilt_config.json` file - [see Tilt Command Options below.](#tilt-command-options)
+7. In a terminal, run `tilt up` at the root of the OpenCost repository. See [Tilt Command Options below for more options.](#options)
+8. Follow instructions in the terminal to open the Tilt UI in browser.
+9. As you update code, `tilt_config.json`, or `tilt-values.yaml`, Tilt will automatically watch relevant source code and recompile, build, deploy, and run tests.
+10. `tilt down` will tear down resources created, otherwise they will continue to run indefinitely.
+11. `kind delete` cluster will delete the local cluster, if desired.
+
+## Tilt Command Options
+
+### via `tilt_config.json`
+The best way to configure OpenCost to use tilt is with a `tilt_config.json` file in the root of the OpenCost repository. The file must live there; a symlink is thebest alternative if you wish to store it elsewhere. This file is automatically reloaded in tilt when updated. Omitted values will use defaults.
```

Comment:
"thebest" should be "the best"

Reply:
Also sometimes you use "Tilt" and other times "tilt", probably stick with "Tilt" everywhere.